### DISCUSSION TITLE
Membuat Virtual Environment

### DISCUSSION THREAD

**Discussion by joejnt** [teamalgoritma/community#135] [Q&A] [answered]
Dear team Algoritma

Saya mendapatkan masalah ketika mencoba untuk membuat Virtual Environtment baru dengan nama 'database_da' namun hasilnya terjadi error. Mohon saran dan petunjuknya.

Terima kasih,

Best regards,
JT

<img width="676" alt="image" src="https://user-images.githubusercontent.com/103427989/173795839-d3d203d0-c82f-4f43-9613-b8edaab2330e.png">

**Comment by inytss** [comment, marked as answer]
Dear @joejnt , 
Sepertinya terdapat kendala koneksi internet pada laptop Anda. Apakah laptop yang Anda gunakan adalah laptop kantor? Jika iya, maka Anda dapat mencoba untuk menghubungi tim IT untuk memberikan akses network dalam mengakses web tertentu.
